testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -544,10 +544,8 @@
   end
   output, _code = $server.run("#{cmd} '#{system_name}' pillar.get '#{key}' #{extra_cmd}")
   STDOUT.puts "#{cmd} '#{system_name}' pillar.get '#{key}' #{extra_cmd} => #{output}"
-  if verb == 'be' && value == ''
-    raise "Output has more than one line: #{output}" unless output.split("\n").length == 1
```

Comment:
Are we sure about removing the case where the `value` is an empty string and the output has more than one line? :thinking:

Reply:
hmmm maybe i need to rethink it, crap